android/source/VideoLocker/src/org/edx/mobile/view/RegisterActivity.java

```diff
@@ -0,0 +1,213 @@
+package org.edx.mobile.view;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.ImageButton;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import org.edx.mobile.R;
+import org.edx.mobile.base.BaseFragmentActivity;
+import org.edx.mobile.model.api.RegisterResponse;
+import org.edx.mobile.model.registration.RegistrationFieldType;
+import org.edx.mobile.model.registration.RegistrationForm;
+import org.edx.mobile.model.registration.RegistrationFormField;
+import org.edx.mobile.module.analytics.ISegment;
+import org.edx.mobile.task.RegisterTask;
+import org.edx.mobile.view.custom.ETextView;
+import org.edx.mobile.view.registration.IRegistrationFieldView;
```

Comment:
Using * for the import might include the classes that we really don't use. Also, Studio  has a feature of improving the imports which changes * import to specific ones. I think, it's good to go with specific imports than *.